packages/contracts/src/predeploys.ts

```diff
@@ -17,5 +17,6 @@ export const predeploys = {
   OVM_L2CrossDomainMessenger: '0x4200000000000000000000000000000000000007',
   Lib_AddressManager: '0x4200000000000000000000000000000000000008',
   OVM_ProxyEOA: '0x4200000000000000000000000000000000000009',
+  OVM_SafetyCache: '0x420000000000000000000000000000000000000c',
```

Comment:
This was chosen because there's a contract at [`...0B` in v0.4.0](https://github.com/ethereum-optimism/optimism/blob/v0.4.0-rc/packages/contracts/src/predeploys.ts#L20). I could have used `...0A`, but it felt conservative to keep incrementing. 🤷‍♂️